rules/scanners-user-agents.data

```diff
@@ -142,6 +142,9 @@ sqlmap
 # sql injection
 # http://sqlninja.sourceforge.net/
 sqlninja
+#sysscan
```

Comment:
IMO you can drop the comments since it's the same.

Reply:
In fact, iit seems like these are used in the rest of the file to indicate the typee of scanner.

Reply:
@fgsch done !!

Reply:
What @csanders-git said. 
My original comment (and @csanders-git one) also applies to 80legs.